source/index.html.md

```diff
@@ -15,6 +15,7 @@ includes:
   - errors
 
 search: true
+#search_timeout: 300
```

Comment:
I will add a simplified version of that. basic logic: when search corpus size > 5000 tokens we will have a 300 delay.